### DISCUSSION TITLE
installing from gitHub

### DISCUSSION THREAD

**Discussion by ghost** [community/community#124282] [New to GitHub]
hi guys, I need to install AASX package explorer from GitHub (https://github.com/eclipse-aaspe/package-explorer). The only problem is that I DONT' KNOW how to do it... Once I download the Zip file there isn't an app or .exe file that I can run... Could someone help me ? (I'm losing my mind)

**Comment by ExpiredWater** [comment]
Hi @roastedDD 

In the readme file there is a link for binaries for windows.
[https://github.com/admin-shell-io/aasx-package-explorer/releases](https://github.com/admin-shell-io/aasx-package-explorer/releases)

https://github.com/eclipse-aaspe/package-explorer/blob/main/README.md
```
Installation
We provide the binaries for Windows 10 in [the releases](https://github.com/admin-shell-io/aasx-package-explorer/releases).

(Remark: In special cases you may like to use a current build. Please click on a green check mark and select "Check-release" details.)
```
